Interface/GIGraph.m

```diff
@@ -313,19 +306,20 @@ - (void)_generateGraph {
   }
   
   // Make sure we have some tips left
-  if (tips.count == 0) {
+  if (tips.commits.count == 0) {
     goto cleanup;
   }
   
   // Re-sort all tips in descending chronological order (this ensures virtual tips will be on the rightside of the tips descending from the same commits)
+  NSMutableArray *tipsArray = tips.commits.mutableCopy;
```

Comment:
This is not efficient: just grab the array and create a sorted copy only if sorting is needed.